core/src/main/java/com/klaytn/caver/tx/kct/KIP7.java

```diff
@@ -0,0 +1,787 @@
+
+    public static final String FUNC_SAFETRANSFERFROM = "safeTransferFrom";
+
+    public static final String FUNC_SUPPORTSINTERFACE = "supportsInterface";
+
+    public static final String FUNC_SYMBOL = "symbol";
+
+    public static final String FUNC_TOTALSUPPLY = "totalSupply";
+
+    public static final String FUNC_TRANSFER = "transfer";
+
+    public static final String FUNC_TRANSFERFROM = "transferFrom";
+
+    public static final String FUNC_UNPAUSE = "unpause";
+
+    public static final Event APPROVAL_EVENT = new Event("Approval",
+            Arrays.<TypeReference<?>>asList(new TypeReference<Address>(true) {}, new TypeReference<Address>(true) {}, new TypeReference<Uint256>() {}));
+    ;
+
+    public static final Event MINTERADDED_EVENT = new Event("MinterAdded",
+            Arrays.<TypeReference<?>>asList(new TypeReference<Address>(true) {}));
+    ;
+
+    public static final Event MINTERREMOVED_EVENT = new Event("MinterRemoved",
+            Arrays.<TypeReference<?>>asList(new TypeReference<Address>(true) {}));
+    ;
+
+    public static final Event PAUSED_EVENT = new Event("Paused",
+            Arrays.<TypeReference<?>>asList(new TypeReference<Address>() {}));
+    ;
+
+    public static final Event PAUSERADDED_EVENT = new Event("PauserAdded",
+            Arrays.<TypeReference<?>>asList(new TypeReference<Address>(true) {}));
+    ;
+
+    public static final Event PAUSERREMOVED_EVENT = new Event("PauserRemoved",
+            Arrays.<TypeReference<?>>asList(new TypeReference<Address>(true) {}));
+    ;
+
+    public static final Event TRANSFER_EVENT = new Event("Transfer",
+            Arrays.<TypeReference<?>>asList(new TypeReference<Address>(true) {}, new TypeReference<Address>(true) {}, new TypeReference<Uint256>() {}));
+    ;
+
+    public static final Event UNPAUSED_EVENT = new Event("Unpaused",
+            Arrays.<TypeReference<?>>asList(new TypeReference<Address>() {}));
+    ;
+
+    protected static final HashMap<String, String> _addresses;
+
+    static {
+        _addresses = new HashMap<String, String>();
+    }
+
+    protected KIP7(String contractAddress, Caver caver, KlayCredentials credentials, int chainId, ContractGasProvider contractGasProvider) {
+        super(BINARY, contractAddress, caver, credentials, chainId, contractGasProvider);
+    }
+
+    protected KIP7(String contractAddress, Caver caver, TransactionManager transactionManager, ContractGasProvider contractGasProvider) {
+        super(BINARY, contractAddress, caver, transactionManager, contractGasProvider);
+    }
+
+    /**
+     * Instance method for getting Event "Approval" in KIP-7 Standard Token
+     *
+     * @param transactionReceipt transactionReceipt after sending SmartContractExecution Transaction to invoke "approve" Method
+     * @return List Object to Approval Event Response
+     */
+    public List<ApprovalEventResponse> getApprovalEvents(KlayTransactionReceipt.TransactionReceipt transactionReceipt) {
```

Comment:
How about adding test cases for the methods that gets the event?
Those methods are public also, so write test code for those will be good.

Reply:
It already contains test code. 
Please check the test code associated with the function that triggers the event.
(ex) transfer(), addMinter(), renounceMinter(), paused()....

Reply:
If this method will be provided to the user as a public method, I think it is better to manage the TC to test this event function separately.

Reply:
Okay, I'll add test code

Reply:
I added Test code.

Reply:
I added comment. Would you check it?